tests/test_content.py

```diff
@@ -31,6 +31,8 @@
     ('top', ('pics/top', '/r/pics', 'top')),
     ('rising', ('r/pics/rising', '/r/pics', 'rising')),
     ('controversial', ('/r/pics/controversial', '/r/pics', 'controversial')),
+# Need a cassette for this, I think
```

Comment:
You are correct that you need to generate cassettes for all of the tests that have changed. Here's the gist how it works:

```
# Install test dependencies
pip install rtv[test]

# Run the test in read-only mode, using the existing cassette
py.test tests/test_content.py::test_content_subreddit_from_name[gilded]

# Run the test using your local reddit credentials, and generate a new cassette
py.test --record-mode=once --refresh-token=~/.local/share/rtv/refresh-token tests/test_content.py::test_content_subreddit_from_name[gilded]
```

Try doing that for all of the tests that you are touching (not the whole test suite) and commit the new cassettes to your branch.

Reply:
I'll try to get this working soon! Thank you.